Formula/gpsbabel.rb

```diff
@@ -18,7 +18,7 @@ class Gpsbabel < Formula
 
   def install
     args = ["--disable-debug", "--disable-dependency-tracking",
-            "--prefix=#{prefix}"]
+            "--prefix=#{prefix}", "CXXFLAGS=-std=c++11"]
```

Comment:
`ENV.cxx11` should be used instead of modifying `args`.

Just call it before `args` is declared and everything should be fine.